README.md

```diff
@@ -8,8 +8,7 @@ Homebrew formula for neovim.
 
 To install the latest release:
 
-    $ brew tap neovim/neovim
-    $ brew install neovim
+    $ brew install neovim/neovim/neovim
```

Comment:
You currently install neovim using two commands:

+ `brew tap neovim/neovim` (`brew tap username/tapname`)
+ `brew intall neovim` (`brew install formulaname`)

The first one taps the repo, while the second one installs the app. However, homebrew is smart about this, so you can simply `brew install neovim/neovim/neovim` (`brew install username/tapname/formulaname`). That single command taps and installs the app in one go.